### PR TITLE
tests: nffs: Tweak Kconfig to fit in RAM

### PR DESCRIPTION
**Pull request by carlescufi**
NFFS tests do not fit in RAM on nRF5x, tweak the configurations so that
they do.

Fixes: #12065

Signed-off-by: Carles Cufi <carles.cufi@nordicsemi.no>

### CODE REVIEW

**Review by galak — CHANGES_REQUESTED** [review]
Can you fix this so it runs on sanitycheck for the nrf boards.  testcase.yaml needs updating.

**Comment by carlescufi** [comment]
@galak done

File: tests/subsys/fs/nffs_fs_api/basic/testcase.yaml [hunk 1]
```diff
@@ -3,4 +3,4 @@ common:
 tests:
   filesystem.nffs.basic:
     extra_args: TEST=basic
-    platform_whitelist: qemu_x86
+    platform_whitelist: qemu_x86 nrf52840_pca10056
```
Comment (galak) [inline tests/subsys/fs/nffs_fs_api/basic/testcase.yaml]: Should this also include: nrf51_pca10028 & nrf52_pca10040?

Looking at tests/subsys/fs/nffs_fs_api/basic/CMakeLists.txt I see:

```
if(BOARD STREQUAL nrf51_pca10028)
    set(CONF_FILE nrf5x.conf)
elseif(BOARD STREQUAL nrf52_pca10040)
    set(CONF_FILE nrf5x.conf)
elseif(BOARD STREQUAL nrf52840_pca10056)
    set(CONF_FILE nrf5x.conf)
endif()
```

Comment (carlescufi) [reply tests/subsys/fs/nffs_fs_api/basic/testcase.yaml]: They won't fit in RAM, the test requires 230KB of RAM and the 52840 is the only one with 256KB

Comment (galak) [reply tests/subsys/fs/nffs_fs_api/basic/testcase.yaml]: > They won't fit in RAM, the test requires 230KB of RAM and the 52840 is the only one with 256KB

So should those be removed from the CMakeLists?

Comment (carlescufi) [reply tests/subsys/fs/nffs_fs_api/basic/testcase.yaml]: I have now

**Comment by carlescufi** [comment]
recheck

**Comment by carlescufi** [comment]
@galak you OK with this now?

**Review by tyrantlizard77 — APPROVED** [review]

**Review by galak — APPROVED** [review]

**Pull request merged**